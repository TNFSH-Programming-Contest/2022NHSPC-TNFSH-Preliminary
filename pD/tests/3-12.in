8 6
2 1 2 1 2 1 1 2
2 2 2 2 1 3
.T.T..
T.....
...TTT
.T.T..
.....T
......
.....T
T..T..
