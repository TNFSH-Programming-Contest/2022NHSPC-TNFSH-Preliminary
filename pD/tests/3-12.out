ATAT..
T...A.
A.ATTT
.T.T.A
.A.A.T
.....A
...A.T
TA.T.A
